packages/react/src/auto/shadcn/ShadcnAutoForm.tsx

```diff
@@ -85,6 +100,39 @@ export const makeAutoForm = <Elements extends ShadcnElements>({
     Button,
   });
 
+  const AutoHasOneForm = makeShadcnAutoHasOneForm({
+    Badge,
+    Button,
+    Command,
+    CommandItem,
+    CommandInput,
+    CommandLoading,
+    DropdownMenu,
+    DropdownMenuContent,
+    DropdownMenuItem,
+    DropdownMenuLabel,
+    DropdownMenuSeparator,
+    DropdownMenuTrigger,
+    Input,
+    Label,
+    CommandList,
+    CommandEmpty,
+    CommandGroup,
+    Checkbox,
+    ScrollArea,
+    Popover,
+    PopoverContent,
+    PopoverTrigger,
+    Dialog,
+    DialogContent,
+    DialogClose,
+    DialogFooter,
+    DialogHeader,
+    DialogTitle,
+    DialogDescription,
+    DialogTrigger,
+  });
```

Comment:
I think it would result in a slightly smaller bundle to just pass `elements` here, and save us the trouble of updating this list all the time 👍